tests/unit/test_circular_imports.py

```diff
@@ -0,0 +1,95 @@
+"""Tests for circular imports in all local packages and modules.
+
+This ensures all internal packages can be imported right away without
+any need to import some other module before doing so.
+
+This module is based on an idea that pytest uses for self-testing:
+* https://github.com/sanitizers/octomachinery/blob/be18b54/tests/circular_imports_test.py
+* https://github.com/pytest-dev/pytest/blob/d18c75b/testing/test_meta.py
+* https://twitter.com/codewithanthony/status/1229445110510735361
+* https://github.com/aio-libs/aiohttp/blob/master/tests/test_circular_imports.py
+"""  # noqa: E501
+import os
+import pkgutil
+import subprocess
+import sys
+from itertools import chain
+from pathlib import Path
+from types import ModuleType
+from typing import Generator
+from typing import List
+
+import pytest
+
+import ansible_navigator
+
+
+def _find_all_importables(pkg: ModuleType) -> List[str]:
+    """Find all importables in the project.
+
+    Return them in order.
+    """
+    return sorted(
+        set(
+            chain.from_iterable(
+                _discover_path_importables(Path(p), pkg.__name__)
+                for p in pkg.__path__  # type: ignore[attr-defined]
```

Comment:
Check if this is still necessary:
```suggestion
                for p in pkg.__path__
```

Reply:
```tests/unit/test_circular_imports.py: note: In function "_find_all_importables":
tests/unit/test_circular_imports.py:36:26: error: Module has no attribute
"__path__"  [attr-defined]
                    for p in pkg.__path__
                             ^
Found 1 error in 1 file (checked 220 source files)
```

Reply:
seems so

Reply:
Looks like updating mypy should fix it: https://github.com/aio-libs/aiohttp/commit/eda8114ae1246aab135cef5201581025b4fb6fd0#diff-6e4d51438e11d25f69b91905d1021363313962f1dc8e251dc1f21580ddc29393L33